dmax.py

```diff
@@ -178,31 +181,43 @@ def main(showid, chosen_season=0, chosen_episode=0, realm=REALMS[0]):
         xls.worksheet.write(xls.row, xls.col(), episode.description)
         xls.worksheet.write(xls.row, xls.col(), filename)
 
-        try:
-            req = get(PLAYER_URL + episode.id, headers={
-                "Authorization": "Bearer " + token,
-                "User-Agent":    USER_AGENT
-            })
-        except Exception as exception:
-            logger.error("Connection for video id {0} failed: {1}".format(episode.id, str(exception)))
-            xls.row += 1
-            xls._col = 0
-            continue
+        fetchAttempts = [1,2,3]
```

Comment:
Please use `for fetchAttempts in range(3)` instead of creating a list

Reply:
I've no real experience with python. I've found the range function yesterday already, but I thought range(1,4,1) is not as comprehendable as [1,2,3] :-)

Reply:
You can directly do a range at the head of the for-loop. Don't worry, I will correct it. Thanks for your fix! :)